Mathlib/Logic/Function/Defs.lean

```diff
@@ -82,6 +82,7 @@ theorem Bijective.comp {g : β → φ} {f : α → β} : Bijective g → Bijecti
   | ⟨h_ginj, h_gsurj⟩, ⟨h_finj, h_fsurj⟩ => ⟨h_ginj.comp h_finj, h_gsurj.comp h_fsurj⟩
 
 /-- `LeftInverse g f` means that `g` is a left inverse to `f`. That is, `g ∘ f = id`. -/
+@[grind ]
```

Comment:
This spurious whitespace is unfortunately required by the `commandStart` linter, due to an underlying problem with the pretty-printing of `@[grind]` syntax.

We could leave as is, and these will eventually be linted once that syntax problem is corrected. Or, in the meantime, someone could ask the `commandStart` linter to relax.